starlite/handlers/http.py

```diff
@@ -211,6 +212,42 @@ async def handler(data: Any, plugins: List["PluginProtocol"], **kwargs: Any) ->
     return handler
 
 
+def _normalize_http_method(http_methods: Union[HttpMethod, Method, List[Union[HttpMethod, Method]]]) -> Set["Method"]:
+    """
+
+    Args:
+        http_methods: A value for http method.
+
+    Returns:
+        A normalized set of http methods.
+    """
+    output: Set[str] = set()
+
+    for method in http_methods if isinstance(http_methods, list) else [http_methods]:
+        if isinstance(method, HttpMethod):
+            output.add(method.value.upper())
+        else:
+            output.add(method.upper())
+
+    return cast("Set[Method]", output)
+
+
+def _get_default_status_code(http_methods: Set["Method"]) -> int:
+    """
+
+    Args:
+        http_methods: A set of Method strings.
+
+    Returns:
+        A status code integer.
+    """
+    if HttpMethod.POST in http_methods:
+        return HTTP_201_CREATED
+    if HttpMethod.DELETE in http_methods:
+        return HTTP_204_NO_CONTENT
+    return HTTP_200_OK
+
+
```

Comment:
Maybe a `handlers.utils` module for these?

Reply:
I dont think we will ever use them outside this module.

Reply:
and in the tests.. not a blocker, just a suggestion.